src/__main__.py

```diff
@@ -1,18 +1,34 @@
 from pathlib import Path
+from pprint import pprint as pp
 import sys
 from .parse_pst import get_email
 
 
 def email_digest(pst_file): 
-    '''function to scan target pst every hour and output a digest of recent activity'''
-    test = get_email(pst_file)
-    print(test)
+    emails = get_email(pst_file)
+    FIELD = 'subject'
+    digest = []
+    for mail in emails:
+        for key in mail:
+            if key in FIELD:
+                message = {}
+                message['sender'] = mail['sender']
+                message['subject'] = mail['subject']
+                message['delivery_time'] = mail['delivery_time']
+                digest.append(message)
+                    
+  
+    for message in digest:
+        result = ' | '.join(map(str, message.values()))
```

Comment:
cool, I do find a list comprehension a bit more readable (instead of map)

would this work?

```
result = " | ".join(val for val in message.values())
```

I think you might not need the str conversion?

Reply:
List comprehension yielded a TypeError, it doesn't like date time

`TypeError: sequence item 2: expected str instance, datetime.datetime found`

Reply:
can you show me the code you had? also can I run this on an existing pst file (in the repo) to provide you a snippet?

Reply:
yes I can :)

```
$ python -m src PDM2.pst
Protein World  | SALE JUST GOT BIGGER!😮 | 2022-07-16 09:02:31
Postmates | Up to 45% off | Consider it a gift 🎁 | 2022-07-16 02:21:01
```

btw can you add a requirements.txt file? it seems one has to install `libpff-python` to get it running ...

Reply:
This works for me:

```
(Pdb) message.values()
dict_values(['Protein World ', 'SALE JUST GOT BIGGER!😮', datetime.datetime(2022, 7, 16, 9, 2, 31)])
(Pdb) " | ".join(message.values())
*** TypeError: sequence item 2: expected str instance, datetime.datetime found
(Pdb) " | ".join(str(val) for val in message.values())
'Protein World  | SALE JUST GOT BIGGER!😮 | 2022-07-16 09:02:31'
```